bin/rustdoc

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+USAGE="Builds Tokio's RustDocs.
+
+USAGE:
+    $(basename "$0") [FLAGS] [<rustdoc-options>...]
+
+FLAGS:
+    -h, --help      Show this help text and exit
+    -u, --unstable  Build documentation for unstable features
+
+NOTES:
+    Additional arguments are passed to the RustDoc invocation. For
+    example:
+
+        $(basename "$0") --open
+
+    will open the documentatiion in a web browser.
+
+    The '-u' or '--unstable' flag will build documentation for unstable
+    features. Unstable features will also be documented if the RUSTFLAGS
+    environment variable contains \"--cfg tokio_unstable\". For example:
+
+        RUSTFLAGS=\"--cfg tokio_unstable\" $(basename "$0")"
+
+set -euo pipefail
+cd "$(dirname "$0")"/..
+
+RUSTDOCFLAGS="${RUSTDOCFLAGS:-} --cfg docsrs"
+
+if echo "${RUSTFLAGS}" | grep -q "tokio_unstable"; then
+    RUSTDOCFLAGS+=" --cfg tokio_unstable"
+fi
+
+declare -a ARGS=()
```

Comment:
```suggestion
ARGS=(--all-features)
```

[It is not easy to properly handle empty arrays in bash](https://stackoverflow.com/questions/7577052/bash-empty-array-expansion-with-set-u/61551944#61551944), so I would suggest including the --all-features flag in ARGS.